gui/CameraView.cxx

```diff
@@ -607,6 +607,13 @@ void CameraView::addFeatureTrack(kwiver::vital::track const& track)
   d->updateFeatures(this);
 }
 
+void CameraView::setFrameName(QString frameName)
+{
+  QTE_D();
+
```

Comment:
superfluous :smile: (...which I wouldn't bother to note except for other more interesting nits)